src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -128,7 +128,8 @@ export function useTaprootInscriptionsInfiniteQuery() {
       const results = responsesArr.flatMap(response => response.results);
 
       // get offset and total from the last response
-      const { offset, total } = responsesArr[responsesArr.length - 1];
+      const offset = responsesArr[responsesArr.length - 1]?.offset;
+      const total = responsesArr[responsesArr.length - 1]?.total;
```

Comment:
not sure that this is necessary after that change with calculated offset, but just in case